execution/types/block_access_list.go

```diff
@@ -220,7 +220,7 @@ func (sc *SlotChanges) DecodeRLP(s *rlp.Stream) error {
 
 func (sc *StorageChange) EncodingSize() int {
 	size := 1 + rlp.IntLenExcludingHead(uint64(sc.Index))
-	size += rlp.StringLen(sc.Value[:])
+	size += sc.Value.ByteLen()
```

Comment:
Should be something like
```
	size++
	size += rlp.Uint256LenExcludingHead(sc.Value)
```
Otherwise it's off by one unless `0 < sc.Value < 128`